oracle/tests/conftest.py

```diff
@@ -55,8 +52,8 @@
         '{}/docker/client/osdt_core.jar:/opt/oracle/instantclient_19_3/osdt_core.jar'.format(HERE),
     ],
     'start_commands': [
-        'bash /tmp/install_instant_client.sh',
-        'apt-get install gcc g++ -y',  # `apt-get update` already ran in install_instant_client.sh
+        'apt-get update',
+        'DEBIAN_FRONTEND=noninteractive apt-get install -yq gcc g++',
```

Comment:
Moving the setting of the `DEBIAN_FRONTEND` env var to `env_vars` below should probably fix the error. I think the way these commands are pre-parsed and run doesn't allow for setting the variable inline.